artemis/placeholder_page_detector.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any
+
+import requests
+
+from artemis import http_requests
+from artemis.config import Config
+
+PLACEHOLDER_PAGE_CONTENT_FILENAME = Config.Modules.PlaceholderPageContent.PLACEHOLDER_PAGE_CONTENT_FILENAME
+
+
+placeholder_page_content = []
```

Comment:
this is constant as well, upcase

Reply:
But after read the file, list will be modified.